pkg/plugins/platform/local_test.go

```diff
@@ -534,6 +519,11 @@ func TestCleanupWithoutCompose(t *testing.T) {
 		assert.False(volume.Labels["com.docker.compose.project"] == "ddev"+strings.ToLower(app.GetName()))
 	}
 
+	// Cleanup the global site database dirs
```

Comment:
I only moved these lines from above (so that they could be done safely, after the containers were shut down.) Checking to see if this is duplicate anyway...